server/data/quotes.json

```diff
@@ -1,4 +1,32 @@
 [
+  {
```

Comment:
Hi @imanishbarnwal, the quotes should be about **learning foreign languages**.

Updated the checklist just so it's clear. Hope you can change the quotes!

Reply:
Ohh okay sure thanks will do it

Reply:
Now check them I just updated all of it.

Reply:
Thanks looks fine now!